rtv/page.py

```diff
@@ -369,11 +416,16 @@ def _add_cursor(self):
     def _remove_cursor(self):
         self._edit_cursor(curses.A_NORMAL)
 
-    def _move_cursor(self, direction):
+    def _move_cursor(self, direction, page_ud=False):
 
         self._remove_cursor()
 
-        valid, redraw = self.nav.move(direction, len(self._subwindows))
+        if page_ud:
+            valid, redraw = self.nav.move_page(direction,
+                                               len(self._subwindows))
```

Comment:
As for a separate issue, what should we do if some post doesn't fit in the window and truncated? In the current implementation, as long as a part of some post is within the window, page up/down skip that post.